test/realtime_barrier_tests.cpp

```diff
@@ -0,0 +1,215 @@
+/*
+ * Copyright (c) 2019, Open Source Robotics Foundation, Inc.
```

Comment:
could touch this up with Colin & Colin Inc

Reply:
Updated with FlyingEinstein.com which is a registered company. I assume the WillowGarage from within the license text remains unchanged.